domains/iamRahul21.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "iamRahul21",
+    "email": "rahuldeka072@gmail.com"
+  },
+  "record": {
+    "CNAME": "iamrahul21.github.io/PortfolioWebsite"
```

```suggestion
    "CNAME": "iamrahul21.github.io"
```
CNAMEs cannot contain slashes `/`.